pyproject.toml

```diff
@@ -16,6 +16,7 @@ duckdb = ">=0.6.0"
 sqlglot = ">=5.1.0"
 altair = ">=4.2.0"
 Jinja2 = ">=3.0.3"
+phonetics = "^1.0.5"
```

Comment:
Sorry, one last thing on this - are we sure we want this as a core import?

It's quite small, but still.

Reply:
As mentioned, we should be able create a binary from a lower-level language and then execute that.

It will be far faster and will remove the dependency.